src/types/config.ts

```diff
@@ -13,6 +13,7 @@ export type ModuleInfoExtended = ModuleInfo & {
 export type NetworkConfig = {
     [networkId: string]: {
         entryPointV6: string;
+        providerURL: string;
```

Comment:
providerURL is to be loaded from env. and can be used while creating contract instance, can we keep it here ? @tomarsachin2271

Reply:
Yeah we can keep it here but it should be array of rpc urls as mentioned in earlier comments.